policyengine_us/variables/gov/states/ri/tax/income/adjusted_gross_income/ri_subtractions.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class ri_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island AGI Subtractions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%201041%20Schedule%20M_w.pdf"
+    defined_for = StateCode.RI
+    adds = [
```

Comment:
we make to make a list parameter (see doc for guidance or other subtractions.yaml files) and the add the parameter here